api/src/middleware/routerTestUtils.ts

```diff
@@ -12,14 +15,26 @@ export const sessionDestroyMock = jest.fn();
 
 export const createAppAgentForRouter = (router: Router): SuperAgentTest => {
   const app = express();
+
+  const server = createServer(app);
+  const io = new Server(server, {
+    cors: {
```

Comment:
The test server doesn't need CORS configuration since all the requests are made by tests, not browsers.

Reply:
Okay, I will remove the CORS configs in this file.